addons/website_sale_mrp/static/tests/tours/test_website_sale_product_availability.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import tourUtils from "@website_sale/js/tours/tour_utils";
+import wTourUtils from "@website/js/tours/tour_utils";
+
+registry.category("web_tour.tours").add("test_website_sale_availability_kit", {
+    test: true,
+    url: "/shop",
+    steps: () => [
+        ...tourUtils.addToCart({ productName: "Component A" }),
+        { trigger: ".availability_messages:contains(99)", run() {} },
```

Comment:
```suggestion
        {
            trigger: ".availability_messages:contains(99)",
            isCheck: true,
        },
```
writing the steps like this makes them a bit easier to follow, and using `isCheck` should provide a reminder in the forward ports to remove them, as it's the default behavior there.